package.json

```diff
@@ -25,6 +26,7 @@
     "gulp-uglify": "^1.5.4",
     "gulp-util": "^3.0.7",
     "merge-stream": "^1.0.0",
+    "prop-types": "^15.5.10",
```

Comment:
- [ ] [Move this to be a dependency](https://github.com/facebook/prop-types#how-to-depend-on-this-package)